New dashboard filters: month and document type

cuducos opened the pull request:
Fix #230 & #231

jtemporal commented:
hey @cuducos any idea why code climate is showing two issues?

jtemporal commented on jarbas/dashboard/admin.py:
```diff
@@ -74,6 +88,40 @@ def queryset(self, request, queryset):
+
+    title = 'mês'
+    parameter_name = 'month'
+    options = ((k, v) for k, v in MONTHS.items())
```
This should be something like `options = tuple((k, v) for k, v in MONTHS.items())` so it is displayed on the dashboard. Right now this filter is not being shown ;)

cuducos commented:
> hey @cuducos any idea why code climate is showing two issues to be fixed?

Repetition in the `admin.py` file. Fixed in 0e6067e (also changed what you've mentioned about generator/tuple) ; )

jtemporal commented:
>also changed what you've mentioned about generator/tuple

I really liked what you did there ;)
